addons/binding/org.openhab.binding.helios/src/main/java/org/openhab/binding/helios/ws/rest/RESTError.java

```diff
@@ -0,0 +1,12 @@
+package org.openhab.binding.helios.ws.rest;
```

Comment:
Every Java file must have a license header. You can run `mvn license:format` on the root of the repo to automatically add missing headers.